### ISSUE TITLE
Process exit with starting from version 1.0 on teamcity

### ISSUE THREAD

**Issue by ZeddEn** [DevExpress/testcafe#3615]
### What is your Test Scenario?
Launching tests via TeamCity using the latest versions (1.0.0, 1.0.1 & 1.1.0)
 
### What is the Current behavior?
Since the version 1.0.0, the builds on TeamCity always exit with error code "Process exited with code -1073741819". This problem was not on version 0.23.x and lower ones. I believe it is a regression.

The problem only occurs via TeamCity builds. No problem locally nor on TeamCity agents (when connecting to the machines and launching the builds)
 
### What is the Expected behavior?
The build should work normally without any error.

The behavior is normal on old versions.
 
### What is your web application and your TestCafe test code?

 
Private website.

<details>
<summary>Your complete test code (or attach your test files):</summary>

 
```js
 
const createTestCafe = require('testcafe');

let testcafe = null;

createTestCafe('localhost', 1337, 1338)
	.then(tc => {
		testcafe = tc;
		const runner = testcafe.createRunner();

		return runner
			.src(['./test/**/*.ts'])
			.browsers(['chrome:headless --no-sandbox'])
			.concurrency(1)
			.reporter('teamcity')
			.screenshots('./report/screenshots/', true, '${USERAGENT}_${FIXTURE}_${TEST}_${QUARANTINE_ATTEMPT}.png')
			.run({
				skipUncaughtErrors: false,
				skipJsErrors: true, 
				quarantineMode: true,
				selectorTimeout: 20000,
				assertionTimeout: 20000
			});
	})
	.then(failedCount => {
		console.log('Tests failed: ' + failedCount);
		testcafe.close();
		process.exit(failedCount ? 1 : 0);
	})
	.catch(err => {
		console.log(err);
		testcafe.close();
		process.exit(1);
	});

```
</details>

<details>
<summary>Your complete test report:</summary>

```

Build 'Autopipeline :: video-showcase :: video-showcase - Functional Tests' #297, branch 'refs/heads/QAT-XXX-Reporting-problem-with-version-1'
Started 2019-03-27 14:32:43 on 'QA61D' by 'Git'
Finished 2019-03-27 14:37:01 with status FAILURE 'Tests failed: 1, passed: 14'
VCS revisions: 'showcase functional tests' (Git): 9254e2b1bfd9d5472cd545ef043c8dd375a0d09e (branch 'refs/heads/QAT-XXX-Reporting-problem-with-version-1')
TeamCity URL http://teamcity.smartadserver.com:80/viewLog.html?buildId=1291679&buildTypeId=Autopipeline_videoshowcase_FunctionalTests 
TeamCity server version is 2017.2.4 (build 51228), server timezone: CET (UTC+01:00)

[14:32:05]E: bt22721 (4m:56s)
[14:32:05]i: TeamCity server version is 2017.2.4 (build 51228)
[14:32:38] : The build is removed from the queue to be prepared for the start
[14:32:38] : Collecting changes in 1 VCS root (1s)
[14:32:38] :	 [Collecting changes in 1 VCS root] VCS Root details
[14:32:38] :		 [VCS Root details] "showcase functional tests" {instance id=6657, parent internal id=4810, parent id=Autopipeline_videoshowcase_ShowcaseFunctionalTests, description: "git@bitbucket.org:smartadserver/showcasefunctionaltests.git#refs/heads/master"}
[14:32:40]i:	 [Collecting changes in 1 VCS root] Detecting changes in VCS root 'showcase functional tests' (used in 'video-showcase - Functional Tests')
[14:32:40]i:	 [Collecting changes in 1 VCS root] Will collect changes for 'showcase functional tests' starting from revision b8a76b08c1f2b206c821c1fc50708f88294b39cf
[14:32:40] :	 [Collecting changes in 1 VCS root] Compute revision for 'showcase functional tests'
[14:32:40] :		 [Compute revision for 'showcase functional tests'] Upper limit revision: 9254e2b1bfd9d5472cd545ef043c8dd375a0d09e
[14:32:40]i:		 [Compute revision for 'showcase functional tests'] MaxModId = 379405
[14:32:40] :		 [Compute revision for 'showcase functional tests'] Latest commit attached to build configuration: 9254e2b1bfd9d5472cd545ef043c8dd375a0d09e
[14:32:40] :		 [Compute revision for 'showcase functional tests'] Computed revision: 9254e2b1bfd9d5472cd545ef043c8dd375a0d09e
[14:32:40] : Starting the build on the agent QA61D
[14:32:43]i: Agent time zone: Europe/Paris
[14:32:43]i: Agent is running under JRE: 1.8.0_66-b17
[14:32:43] : Clearing temporary directory: C:\BuildAgent\temp\buildTmp
[14:32:43] : Publishing internal artifacts
[14:32:43] :	 [Publishing internal artifacts] Publishing 1 file using [WebPublisher]
[14:32:43] :	 [Publishing internal artifacts] Publishing 1 file using [ArtifactsCachePublisher]
[14:32:43] : Using vcs information from agent file: 8fef817816ff846b.xml
[14:32:43] : Checkout directory: C:\BuildAgent\work\8fef817816ff846b
[14:32:43] : Updating sources: auto checkout (on agent) (7s)
[14:32:43] :	 [Updating sources] Will use agent side checkout
[14:32:43] :	 [Updating sources] VCS Root: showcase functional tests (7s)
[14:32:43] :		 [VCS Root: showcase functional tests] revision: 9254e2b1bfd9d5472cd545ef043c8dd375a0d09e
[14:32:43] :		 [VCS Root: showcase functional tests] Git version: 2.12.2.0
[14:32:43] :		 [VCS Root: showcase functional tests] Update git mirror (C:\BuildAgent\system\git\git-F8281EFA.git) (4s)
[14:32:43] :			 [Update git mirror (C:\BuildAgent\system\git\git-F8281EFA.git)] "C:\Program Files\Git\bin\git.exe" show-ref
[14:32:43] :			 [Update git mirror (C:\BuildAgent\system\git\git-F8281EFA.git)] "C:\Program Files\Git\bin\git.exe" -c credential.helper= ls-remote origin
[14:32:45] :			 [Update git mirror (C:\BuildAgent\system\git\git-F8281EFA.git)] "C:\Program Files\Git\bin\git.exe" show-ref refs/remotes/origin/QAT-XXX-Reporting-problem-with-version-1
[14:32:45] :			 [Update git mirror (C:\BuildAgent\system\git\git-F8281EFA.git)] "C:\Program Files\Git\bin\git.exe" show-ref refs/heads/QAT-XXX-Reporting-problem-with-version-1
[14:32:46] :			 [Update git mirror (C:\BuildAgent\system\git\git-F8281EFA.git)] "C:\Program Files\Git\bin\git.exe" -c credential.helper= fetch --progress origin +refs/heads/QAT-XXX-Reporting-problem-with-version-1:refs/heads/QAT-XXX-Reporting-problem-with-version-1 (2s)
[14:32:48]i:				 ["C:\Program Files\Git\bin\git.exe" -c credential.helper= fetch --progress origin +refs/heads/QAT-XXX-Reporting-problem-with-version-1:refs/heads/QAT-XXX-Reporting-problem-with-version-1] remote: Counting objects: 4, done.        
[14:32:48]i:				 ["C:\Program Files\Git\bin\git.exe" -c credential.helper= fetch --progress origin +refs/heads/QAT-XXX-Reporting-problem-with-version-1:refs/heads/QAT-XXX-Reporting-problem-with-version-1] remote: Compressing objects:  25% (1/4)           
[14:32:48]i:				 ["C:\Program Files\Git\bin\git.exe" -c credential.helper= fetch --progress origin +refs/heads/QAT-XXX-Reporting-problem-with-version-1:refs/heads/QAT-XXX-Reporting-problem-with-version-1] remote: Compressing objects:  50% (2/4)           
[14:32:48]i:				 ["C:\Program Files\Git\bin\git.exe" -c credential.helper= fetch --progress origin +refs/heads/QAT-XXX-Reporting-problem-with-version-1:refs/heads/QAT-XXX-Reporting-problem-with-version-1] remote: Compressing objects:  75% (3/4)           
[14:32:48]i:				 ["C:\Program Files\Git\bin\git.exe" -c credential.helper= fetch --progress origin +refs/heads/QAT-XXX-Reporting-problem-with-version-1:refs/heads/QAT-XXX-Reporting-problem-with-version-1] remote: Compressing objects: 100% (4/4)           
[14:32:48]i:				 ["C:\Program Files\Git\bin\git.exe" -c credential.helper= fetch --progress origin +refs/heads/QAT-XXX-Reporting-problem-with-version-1:refs/heads/QAT-XXX-Reporting-problem-with-version-1] remote: Compressing objects: 100% (4/4), done.        
[14:32:48]i:				 ["C:\Program Files\Git\bin\git.exe" -c credential.helper= fetch --progress origin +refs/heads/QAT-XXX-Reporting-problem-with-version-1:refs/heads/QAT-XXX-Reporting-problem-with-version-1] remote: Total 4 (delta 3), reused 0 (delta 0)        
[14:32:48]i:				 ["C:\Program Files\Git\bin\git.exe" -c credential.helper= fetch --progress origin +refs/heads/QAT-XXX-Reporting-problem-with-version-1:refs/heads/QAT-XXX-Reporting-problem-with-version-1] From bitbucket.org:smartadserver/showcasefunctionaltests
[14:32:48]i:				 ["C:\Program Files\Git\bin\git.exe" -c credential.helper= fetch --progress origin +refs/heads/QAT-XXX-Reporting-problem-with-version-1:refs/heads/QAT-XXX-Reporting-problem-with-version-1]    ab73889..9254e2b  QAT-XXX-Reporting-problem-with-version-1 -> QAT-XXX-Reporting-problem-with-version-1
[14:32:48]i:				 ["C:\Program Files\Git\bin\git.exe" -c credential.helper= fetch --progress origin +refs/heads/QAT-XXX-Reporting-problem-with-version-1:refs/heads/QAT-XXX-Reporting-problem-with-version-1]    ab73889..9254e2b  QAT-XXX-Reporting-problem-with-version-1 -> origin/QAT-XXX-Reporting-problem-with-version-1
[14:32:48] :			 [Update git mirror (C:\BuildAgent\system\git\git-F8281EFA.git)] "C:\Program Files\Git\bin\git.exe" log -n1 --pretty=format:%H%x20%s 9254e2b1bfd9d5472cd545ef043c8dd375a0d09e --
[14:32:48] :			 [Update git mirror (C:\BuildAgent\system\git\git-F8281EFA.git)] "C:\Program Files\Git\bin\git.exe" pack-refs --all
[14:32:48] :		 [VCS Root: showcase functional tests] Update checkout directory (C:\BuildAgent\work\8fef817816ff846b) (2s)
[14:32:48] :			 [Update checkout directory (C:\BuildAgent\work\8fef817816ff846b)] "C:\Program Files\Git\bin\git.exe" config lfs.storage C:\BuildAgent\system\git\git-F8281EFA.git\lfs
[14:32:48] :			 [Update checkout directory (C:\BuildAgent\work\8fef817816ff846b)] "C:\Program Files\Git\bin\git.exe" config core.sparseCheckout true
[14:32:48] :			 [Update checkout directory (C:\BuildAgent\work\8fef817816ff846b)] "C:\Program Files\Git\bin\git.exe" show-ref
[14:32:48] :			 [Update checkout directory (C:\BuildAgent\work\8fef817816ff846b)] "C:\Program Files\Git\bin\git.exe" show-ref refs/remotes/origin/QAT-XXX-Reporting-problem-with-version-1
[14:32:48] :			 [Update checkout directory (C:\BuildAgent\work\8fef817816ff846b)] "C:\Program Files\Git\bin\git.exe" log -n1 --pretty=format:%H%x20%s 9254e2b1bfd9d5472cd545ef043c8dd375a0d09e --
[14:32:48] :			 [Update checkout directory (C:\BuildAgent\work\8fef817816ff846b)] "C:\Program Files\Git\bin\git.exe" branch
[14:32:48] :			 [Update checkout directory (C:\BuildAgent\work\8fef817816ff846b)] "C:\Program Files\Git\bin\git.exe" -c credential.helper= reset --hard 9254e2b1bfd9d5472cd545ef043c8dd375a0d09e
[14:32:48] :			 [Update checkout directory (C:\BuildAgent\work\8fef817816ff846b)] "C:\Program Files\Git\bin\git.exe" branch --set-upstream-to=refs/remotes/origin/QAT-XXX-Reporting-problem-with-version-1
[14:32:48] :			 [Update checkout directory (C:\BuildAgent\work\8fef817816ff846b)] Cleaning showcase functional tests in C:\BuildAgent\work\8fef817816ff846b the file set ALL_UNTRACKED
[14:32:48] :			 [Update checkout directory (C:\BuildAgent\work\8fef817816ff846b)] "C:\Program Files\Git\bin\git.exe" clean -f -d -x
[14:32:51] : Build preparation done
[14:32:51] : Step 1/5: Change node version (Command Line)
[14:32:51] :	 [Step 1/5] Starting: C:\BuildAgent\temp\agentTmp\custom_script311161702191360677.cmd
[14:32:51] :	 [Step 1/5] in directory: C:\BuildAgent\work\8fef817816ff846b
[14:32:51] :	 [Step 1/5] PATH -= C:\Program Files\nodejs
[14:32:51] :	 [Step 1/5] PATH += C:\ProgramData\nvs\node\8.11.3\x64
[14:32:51] :	 [Step 1/5] Process exited with code 0
[14:32:51]W: Step 2/5: Installing Node Modules (Command Line) (18s)
[14:32:51]i:	 [Step 2/5] Launching "npm" in working directory C:\BuildAgent\work\8fef817816ff846b as it is not present in checkout directory C:\BuildAgent\work\8fef817816ff846b
[14:32:51]i:	 [Step 2/5] Running interpreter: C:\Windows\system32\cmd.exe
[14:32:51] :	 [Step 2/5] Starting: C:\Windows\system32\cmd.exe /c npm install
[14:32:51] :	 [Step 2/5] in directory: C:\BuildAgent\work\8fef817816ff846b
[14:33:09]W:	 [Step 2/5] npm WARN adplayer-test@1.0.0 No repository field.
[14:33:09]W:	 [Step 2/5] 
[14:33:09] :	 [Step 2/5] added 338 packages from 254 contributors and audited 3270 packages in 16.815s
[14:33:09] :	 [Step 2/5] found 0 vulnerabilities
[14:33:09] :	 [Step 2/5] 
[14:33:09] :	 [Step 2/5] Process exited with code 0
[14:33:09]E: Step 3/5: Run Tests (Command Line) (3m:46s)
[14:33:10] :	 [Step 3/5] Starting: C:\BuildAgent\temp\agentTmp\custom_script1369347623203828631.cmd
[14:33:10] :	 [Step 3/5] in directory: C:\BuildAgent\work\8fef817816ff846b
[14:33:16] :	 [Step 3/5] Starting test run!
[14:33:16] :	 [Step 3/5]     Start Time: Wed Mar 27 2019 14:33:16 GMT+0100 (Romance Standard Time)
[14:33:16] :	 [Step 3/5]     User Agents: HeadlessChrome 73.0.3683 / Windows 10.0.0
[14:33:16] :	 [Step 3/5]     Test Count: 218
[14:33:16] :	 [Step 3/5] Direct Plugin Overlay Placement Data (1m:44s)
[14:33:30] :		 [Direct Plugin Overlay Placement Data] Checking Ad is Present - Default Values (Flowplayer - Latest)
[14:33:40] :		 [Direct Plugin Overlay Placement Data] Checking Ad is Not Present - Incorrect Values (Flowplayer - Latest)
[14:33:50] :		 [Direct Plugin Overlay Placement Data] Checking Ad is Present - Default Values (Flowplayer - Second to Last)
[14:34:00] :		 [Direct Plugin Overlay Placement Data] Checking Ad is Not Present - Incorrect Values (Flowplayer - Second to Last)
[14:34:11] :		 [Direct Plugin Overlay Placement Data] Checking Ad is Present - Default Values (JW Player - Latest)
[14:34:22] :		 [Direct Plugin Overlay Placement Data] Checking Ad is Not Present - Incorrect Values (JW Player - Latest)
[14:34:32] :		 [Direct Plugin Overlay Placement Data] Checking Ad is Present - Default Values (JW Player - Second to Last)
[14:34:41] :		 [Direct Plugin Overlay Placement Data] Checking Ad is Not Present - Incorrect Values (JW Player - Second to Last)
[14:34:51] :		 [Direct Plugin Overlay Placement Data] Checking Ad is Present - Default Values (Simple Video tag - )
[14:35:01] :		 [Direct Plugin Overlay Placement Data] Checking Ad is Not Present - Incorrect Values (Simple Video tag - )
[14:35:01]F:	 [Step 3/5] Direct Plugin Preroll Display (1m:54s)
[14:35:04] :		 [Direct Plugin Preroll Display] Checking Ad is Present (Flowplayer - Latest)
[14:35:14] :		 [Direct Plugin Preroll Display] Checking Main Content Resume After the Ad Ends (Flowplayer - Latest)
[14:35:22] :		 [Direct Plugin Preroll Display] Checking Main Content Resume After Skipping the Ad (Flowplayer - Latest)
[14:35:32] :		 [Direct Plugin Preroll Display] Default Pixel Verification (Flowplayer - Latest)
[14:36:54]F:		 [Direct Plugin Preroll Display] Pause and Play Pixel Verification (Flowplayer - Latest)
[14:36:54]F:			 [Pause and Play Pixel Verification (Flowplayer - Latest)] Test Failed
[14:36:54]F:			 [Pause and Play Pixel Verification (Flowplayer - Latest)] 
Cannot obtain information about the node because the specified selector does not match any node in the DOM tree.

Â > | Selector('#container div.sas-reset > div.sas-btn-play...)

Browser: HeadlessChrome 73.0.3683 / Windows 10.0.0
Screenshot: C:\BuildAgent\work\8fef817816ff846b\report\screenshots\HeadlessChrome_73.0.3683_Windows_10.0.0_Direct Plugin Preroll Display_Pause and Play Pixel Verification (Flowplayer - Latest)_3.png

   149 |   * @param {TestController} t
   150 |   * @memberof PlayerPage
   151 |   */
   152 |    public async pauseAdAction(t: TestController) {
   153 |        await t
 > 154 |            .expect(this.pauseAd.visible).ok()
   155 |            .click(this.pauseAd)
   156 |            .expect(this.playAd.visible).ok();
   157 |    }
   158 |    /**
   159 |    * Mute Ad Action

   at PlayerPage.<anonymous> (C:\BuildAgent\work\8fef817816ff846b\model\playerPage.ts:154:43)
   at <anonymous> (C:\BuildAgent\work\8fef817816ff846b\model\playerPage.ts:7:71)
   at __awaiter (C:\BuildAgent\work\8fef817816ff846b\model\playerPage.ts:3:12)
   at PlayerPage.pauseAdAction (C:\BuildAgent\work\8fef817816ff846b\model\playerPage.ts:152:16)
   at Object.<anonymous> (C:\BuildAgent\work\8fef817816ff846b\test\plugin\DirectPrerollAndPixelsTest.ts:97:30)
   at fulfilled (C:\BuildAgent\work\8fef817816ff846b\test\plugin\DirectPrerollAndPixelsTest.ts:4:58)


[14:36:54]E:	 [Step 3/5] Failed tests detected
[14:36:55]W:		 [Direct Plugin Preroll Display] Process exited with code -1073741819
[14:36:55]E:	 [Step 3/5] Process exited with code -1073741819
[14:36:56]E:	 [Step 3/5] Step Run Tests (Command Line) failed
[14:36:56] : Step 4/5: Allure Report (Allure Report) (3s)
[14:36:56] :	 [Step 4/5] http://teamcity.smartadserver.com:80//repository/downloadAll/Autopipeline_videoshowcase_FunctionalTests/.lastFinished/artifacts.zip?branch=refs/heads/QAT-XXX-Reporting-problem-with-version-1
[14:36:57] :	 [Step 4/5] Program environment variables: {=C:=c:\BuildAgent\bin, ALLUSERSPROFILE=C:\ProgramData, APPDATA=C:\Users\http_user\AppData\Roaming, BUILD_NUMBER=297, BUILD_VCS_NUMBER=9254e2b1bfd9d5472cd545ef043c8dd375a0d09e, BUILD_VCS_NUMBER_Autopipeline_videoshowcase_ShowcaseFunctionalTests=9254e2b1bfd9d5472cd545ef043c8dd375a0d09e, COMPUTERNAME=QA61D, ChocolateyInstall=C:\ProgramData\chocolatey, ChocolateyLastPathUpdate=mar. août  8 16:21:01 2017, ComSpec=C:\Windows\system32\cmd.exe, CommonProgramFiles=C:\Program Files (x86)\Common Files, CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files, CommonProgramW6432=C:\Program Files\Common Files, DC_ID=3, ERLANG_HOME=C:\Program Files\Erlang OTP\erl8.3, FLEX_HOME=C:\Program Files (x86)\Flex\Flex-4.16.0, GRADLE_HOME=C:\Program Files (x86)\Gradle\gradle-4.10.3, GYP_MSVS_VERSION=2015, HOMEDRIVE=C:, HOMEPATH=\Users\http_user, JAVA_HOME=C:\Program Files\Java\jdk1.8.0_191, JDK_17=C:\Program Files\Java\jdk1.7.0_80, JDK_17_x64=C:\Program Files\Java\jdk1.7.0_80, JDK_18=C:\Program Files\Java\jdk1.8.0_191, JDK_18_x64=C:\Program Files\Java\jdk1.8.0_191, JDK_HOME=C:\Program Files\Java\jdk1.8.0_191, JRE_HOME=C:\Program Files\Java\jdk1.8.0_191, LOCALAPPDATA=C:\Users\http_user\AppData\Local, LOGONSERVER=\\BDC2003D, M2_HOME=C:\Program Files (x86)\Maven\apache-maven-3.5.0, NUMBER_OF_PROCESSORS=8, OS=Windows_NT, PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC, PROCESSOR_ARCHITECTURE=x86, PROCESSOR_ARCHITEW6432=AMD64, PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 79 Stepping 1, GenuineIntel, PROCESSOR_LEVEL=6, PROCESSOR_REVISION=4f01, PSModulePath=%ProgramFiles%\WindowsPowerShell\Modules;C:\Windows\system32\WindowsPowerShell\v1.0\Modules, PUBLIC=C:\Users\Public, Path=C:\Program Files (x86)\Ant\apache-ant-1.10.1\bin;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\Program Files (x86)\Flex\Flex-4.16.0\bin;C:\Program Files (x86)\Gradle\gradle-4.10.3\bin;C:\Program Files (x86)\Maven\apache-maven-3.5.0\bin;C:\Program Files (x86)\Nuget;C:\Program Files (x86)\Yarn\bin;C:\Program Files\dotnet\;C:\Program Files\Git\cmd;C:\Program Files\Java\jdk1.8.0_191\bin;C:\Program Files\Java\jre1.8.0_191\bin;C:\Program Files\Python 2.7\;C:\Program Files\Python 2.7\Scripts;C:\Program Files\Sonar\sonar-scanner-3.2.0.1227-windows\bin;C:\Program Files\Sonar\sonar-scanner-msbuild-4.4.2.1543-net46;C:\Program Files\TortoiseSVN\bin;C:\ProgramData\chocolatey\bin;C:\ProgramData\Oracle\Java\javapath;C:\Tools\DuplicateKeyFinder;C:\Tools\PuTTY\;C:\Users\http_user\AppData\Local\Microsoft\WindowsApps;C:\Users\http_user\AppData\Roaming\npm;C:\Windows;C:\Windows\system32;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin;C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\CommonExtensions\Microsoft\TestWindow;C:\Program Files\nodejs;C:\ProgramData\nvs\;C:\Program Files (x86)\OpenCover;C:\Program Files\Microsoft VS Code\bin;C:\Users\http_user\AppData\Local\Yarn\bin;C:\Users\http_user\.dotnet\tools, ProgramData=C:\ProgramData, ProgramFiles=C:\Program Files (x86), ProgramFiles(x86)=C:\Program Files (x86), ProgramW6432=C:\Program Files, SONAR_RUNNER_HOME=C:\Program Files\Sonar\sonar-scanner-3.2.0.1227-windows, SystemDrive=C:, SystemRoot=C:\Windows, TEAMCITY_BUILDCONF_NAME=video-showcase - Functional Tests, TEAMCITY_BUILD_PROPERTIES_FILE=C:\BuildAgent\temp\buildTmp\teamcity.build5931855730434081710.properties, TEAMCITY_CAPTURE_ENV="c:\BuildAgent\jre\bin\java.exe" -jar "C:\BuildAgent\plugins\environment-fetcher\bin\env-fetcher.jar", TEAMCITY_GIT_PATH=C:\Program Files\Git\bin\git.exe, TEAMCITY_JRE=C:\Program Files\Java\jre1.8.0_171, TEAMCITY_PROJECT_NAME=video-showcase, TEAMCITY_VERSION=2017.2.4 (build 51228), TEMP=C:\BuildAgent\temp\buildTmp, TMP=C:\BuildAgent\temp\buildTmp, TMPDIR=C:\BuildAgent\temp\buildTmp, USERDNSDOMAIN=AUFEMININ.COM, USERDOMAIN=AUFEMININ2K, USERDOMAIN_ROAMINGPROFILE=AUFEMININ2K, USERNAME=http_user, USERPROFILE=C:\Users\http_user, VS140COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\, WRAPPER_ARCH=x86, WRAPPER_BITS=32, WRAPPER_FILE_SEPARATOR=\, WRAPPER_OS=windows, WRAPPER_PATH_SEPARATOR=;, windir=C:\Windows}
[14:36:57] :	 [Step 4/5] Program working directory: C:\BuildAgent\work\8fef817816ff846b
[14:36:57] :	 [Step 4/5] Program path: C:\BuildAgent\tools\allure-commandline\bin\allure.bat
[14:36:57] :	 [Step 4/5] Program args: [generate, C:\BuildAgent\work\8fef817816ff846b\allure\allure-results, -o, C:\BuildAgent\work\8fef817816ff846b\allure\allure-report]
[14:36:57] :	 [Step 4/5] Starting: C:\BuildAgent\tools\allure-commandline\bin\allure.bat generate C:\BuildAgent\work\8fef817816ff846b\allure\allure-results -o C:\BuildAgent\work\8fef817816ff846b\allure\allure-report
[14:36:57] :	 [Step 4/5] in directory: C:\BuildAgent\work\8fef817816ff846b
[14:36:59] :	 [Step 4/5] Report successfully generated to C:\BuildAgent\work\8fef817816ff846b\allure\allure-report
[14:36:59] :	 [Step 4/5] Process exited with code 0
[14:36:59] :	 [Step 4/5] Publishing artifacts
[14:36:59] :		 [Publishing artifacts] Collecting files to publish: [C:\BuildAgent\work\8fef817816ff846b\allure\allure-report]
[14:36:59] :		 [Publishing artifacts] Publishing 38 files using [WebPublisher]: allure/allure-report
[14:36:59] :		 [Publishing artifacts] Publishing 38 files using [ArtifactsCachePublisher]: allure/allure-report
[14:37:00] : Step 5/5: Close remaining browser instances (Command Line)
[14:37:00] :	 [Step 5/5] Starting: C:\BuildAgent\temp\agentTmp\custom_script2312079483963776662.cmd
[14:37:00] :	 [Step 5/5] in directory: C:\BuildAgent\work\8fef817816ff846b
[14:37:00] :	 [Step 5/5] process "chrome.exe" not running.
[14:37:00] :	 [Step 5/5] process "firefox.exe" not running.
[14:37:00] :	 [Step 5/5] Process exited with code 0
[14:37:01] : Publishing internal artifacts
[14:37:01] :	 [Publishing internal artifacts] Publishing 1 file using [WebPublisher]
[14:37:01] :	 [Publishing internal artifacts] Publishing 1 file using [ArtifactsCachePublisher]
[14:37:01] : Publishing artifacts
[14:37:01] :	 [Publishing artifacts] Collecting files to publish: [/report/screenshots => screenshots.zip]
[14:37:01] :	 [Publishing artifacts] Creating archive screenshots.zip
[14:37:01] :		 [Creating archive screenshots.zip] Creating C:\BuildAgent\temp\buildTmp\ZipPreprocessor7976591417775052024\screenshots.zip
[14:37:01] :		 [Creating archive screenshots.zip] Archive was created, file size 1.18 MB (1238013 bytes)
[14:37:01] :	 [Publishing artifacts] Publishing 1 file using [WebPublisher]: /report/screenshots => screenshots.zip
[14:37:01] :	 [Publishing artifacts] Publishing 1 file using [ArtifactsCachePublisher]: /report/screenshots => screenshots.zip
[14:37:01]E: 1 failed test detected
[14:37:02] : Build finished



```
</details>

### Steps to Reproduce:

See description above.
 
### Your Environment details:
 
* testcafe version:                   Starting from 1.0.0
* node.js version:                    8.11.3
* command-line arguments:  node ./conf/runner.js  (runner content on details)
* browser name and version: Tested on both Firefox and Chrome
* platform and version:          Windowe 10
* other:

**Comment by miherlosev** [comment]
Hi @ZeddEn 

I see in your report the follows error message and stack trace:

```js
Cannot obtain information about the node because the specified selector does not match any node in the DOM tree.

Â > | Selector('#container div.sas-reset > div.sas-btn-play...)

 149 |   * @param {TestController} t
   150 |   * @memberof PlayerPage
   151 |   */
   152 |    public async pauseAdAction(t: TestController) {
   153 |        await t
 > 154 |            .expect(this.pauseAd.visible).ok()
   155 |            .click(this.pauseAd)
   156 |            .expect(this.playAd.visible).ok();
   157 |    }
   158 |    /**
   159 |    * Mute Ad Action
```

Please ensure that your test is correct and there is a visible element that can be found by Selector('#container div.sas-reset > div.sas-btn-play...).

**Comment by ZeddEn** [comment]
Hello,
Sorry but that is not the problem. Even if the tests do not fail I still get the Process exited with code -1073741819
That was just one sample of our many builds of different projects with the same error.

**Comment by miherlosev** [comment]
I searched information about the `1073741819` error in Google and all found articles point to the problem with the sound driver. This issue doesn't seem to be related to TestCafe.
So, I am afraid this information is not sufficient to find the cause of the issue. 
Could you please provide a page or small example so that we can reproduce it on our side?

**Comment by ZeddEn** [comment]
Sadly everything is in our intranet and there is no public access...

**Comment by AlexKamaev** [comment]
@ZeddEn 
The issue is hardly related to TestCafe. I would recommend you ask the question on StackOverflow with the `teamcity` tag. Also, there are some useful posts on StackOverflow: https://stackoverflow.com/questions/771857/nant-build-fails-but-only-in-teamcity/958981#958981

**Comment by ZeddEn** [comment]
Hello, I have found the source of the problem and it is in testcafe level: after removing "quarantineMode: true," from the runner the tests worked without a problem. So the retry mechanism is what caused the build to exit with that code. It looks like a regression with me because it started only after version 1.0.0 (no problem previously back on 0.23.3 and before it). 

Btw, I have changed pretty much everything in TeamCity settings (user with admin rigths, no sounds, etc) and it did not change anything

**Comment by AndreyBelym** [comment]
I don't know which changes between `0.23.3` and `1.0.0` could cause this behavior. If you can't provide an example project, could please try to find the commit that causes the problem? You can fork and clone the TestCafe repository and perform the following steps for each commit from [this list](https://github.com/DevExpress/testcafe/compare/v0.23.3...DevExpress:v1.0.0):
1. `git checkout ${Commit ID}`
2. `npm i`
3. `npx gulp fast-build`
4. `npm pack`
5. Rename the `testcafe-0.23.3.tgz` archive generated by the `npm pack` to `testcafe-0.23.3-${Commit ID}.tgz`.
6. Create a GitHub release in your fork and upload the  `testcafe-0.23.3-${Commit ID}.tgz` archive to the created release. Take a look at this release in my fork for a reference: https://github.com/AndreyBelym/testcafe/releases/tag/gh-3114-4
7. Replace the TestCafe version in your `package.json` with a link to the uploaded archive and run your builds. Report back the commit ID that can be used to reproduce the issue.

**Comment by ZeddEn** [comment]
Hello,
I have followed your steps and tested with many versions from the commit list:

- 1.0.0-alpha.1-e2bdb7c 
- 1.0.0-alpha.1-16df208 
- 0.23.3-b60792b 
- 0.23.3-89ee178 
- 0.23.0-9bdf499 

None of them seems to work as I get always the same error on my teamcity builds (even with the first commit on the list '89ee178')
Here's the list of the tags: https://github.com/ZeddEn/testcafe/tags

So basically, it works without any problem with the release 0.23.3 but does not with the commit just after it "0.23.3-89ee178" which is a bit weird. Has there no commit or release between the two versions?

**Comment by AlexKamaev** [comment]
As you discovered there were no any significant changes between `v.0.23.3 - 631463d` and `89ee178` versions, so I don't think that the cause of the issue lies here.
You mentioned that if you turn off the `quarantine mode` the tests work without error, so I would recommend you research in this direction. Please refer to the file history to see possible changes of the quarantine mechanism https://github.com/DevExpress/testcafe/blame/28925089a94207ebaf45ccebd77970a35b2d4934/src/runner/test-run-controller.js
Also I would recommend you double check this commit 
`Implemented TestCafe configuration file (#3147) - 0.23.3-b60792b` because it's only one significant commit which could lead to the issue in the described time.
